elastio-stream-kafka/README.md

```diff
@@ -0,0 +1,60 @@
+# Elastio Stream (MSK)Kafka <!--Discuss another naming-->
+# Quick start
+## Create machine:
+1. Create an EC2 instance( Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type, instance type: t2.xlagre) in the same VPC as your MSK cluster.
```

Comment:
Don't push people to use `t2` instance types, they are slow and expensive.  Prefer `t3`.  Also you mis-spelled "xlarge"

```suggestion
1. Create an EC2 instance( Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type, instance type: t3.xlarge) in the same VPC as your MSK cluster.
```